src/allmydata/test/common_util.py

```diff
@@ -70,10 +77,11 @@ def flip_bit(good, which):
 def flip_one_bit(s, offset=0, size=None):
     """ flip one random bit of the string s, in a byte greater than or equal to offset and less
     than offset+size. """
+    assert isinstance(s, binary_type)
     if size is None:
         size=len(s)-offset
     i = randrange(offset, offset+size)
-    result = s[:i] + chr(ord(s[i])^(0x01<<randrange(0, 8))) + s[i+1:]
+    result = s[:i] + bchr(bord(s[i])^(0x01<<randrange(0, 8))) + s[i+1:]
```

Comment:
I don't understand why we want to call `chr(ord())` at all, since they are inverses of each other—the output is guaranteed identical to `s[i]`, no? 🤔

Reply:
I've decided not to care. I don't want to change logic, I want to port to Python 3.